Automattic/jetpack
# Viewport lib: switch from internal component to package from Calypso monorepo

#### Changes proposed in this Pull Request:

Until now, we've used an internal `viewport` lib (previously a dops component) to handle things like `isMobile` and `getWindowInnerWidth`. 

That lib was originally a copy from a lib also used in Calypso.

While this worked well, it meant that we did not benefit from enhancements and bug fixes that were made in the Calypso version.

From now on, we'll benefit from those changes because we'll use the same version as the one in Calypso, via an npm package.

This also means that we could use this in Gutenberg blocks in the future. Reference:
https://github.com/Automattic/jetpack/pull/13999#discussion_r378226884

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* It does not add any new feature. Internal discussion: p1581583668341200-slack-team-calypso

#### Testing instructions:

* Go to Jetpack > Settings on a mobile device
* Ensure that things like switching between tabs and searching still work.

#### Proposed changelog entry for your changes:

* N/A
